packages/config/src/projects/project/Project.ts

```diff
@@ -11,18 +13,60 @@ export interface Project {
   id: ProjectId
   slug: string
   name: string
+  shortName: string | undefined
   addedAt: UnixTime
   // data
+  statuses?: ProjectStatuses
+  scalingInfo?: ProjectScalingInfo
+  scalingRisks?: ProjectScalingRisks
   proofVerification?: ProofVerification
   daBridges?: (OnChainDaBridge | EnshrinedBridge | NoDaBridge | DacBridge)[]
+  countdowns?: ProjectCountdowns
   // tags
   isBridge?: true
-  isLayer2?: true
-  isLayer3?: true
   isScaling?: true
   isZkCatalog?: true
   isDaLayer?: true
   isUpcoming?: true
-  isUnderReview?: true
   isArchived?: true
 }
+
+export interface ProjectStatuses {
+  yellowWarning: string | undefined
+  redWarning: string | undefined
+  isUnderReview: boolean
+  isUnverified: boolean
+}
+
+export interface ProjectScalingInfo {
+  layer: 'layer2' | 'layer3'
+  /** e.g. Optimistic Rollup */
+  type: string
+  /** In the future this will be reflected as `type === 'Other'` */
+  isOther: boolean
+  hostChain: {
+    id: ProjectId
+    slug: string
+    name: string
+    shortName: string | undefined
+  }
+  stack: string | undefined
+  raas: string | undefined
+  daLayer: string
+  stage: string
+  purposes: string[]
+}
+
+export interface ProjectScalingRisks {
+  self: ScalingProjectRiskView
+  host: ScalingProjectRiskView | undefined
+  stacked: ScalingProjectRiskView | undefined
+}
```

Comment:
maybe better to use union?
```suggestion
export type ProjectScalingRisks = {
  type: 'single'
  self: ScalingProjectRiskView
} | {
  type: 'multiple'
  host: ScalingProjectRiskView
  stacked: ScalingProjectRiskView
}
```

Reply:
I errored on the ease of use. Also when I was writing there was this edge case of a project having stacked risks but no host risks. After removing hostChain = multiple this is no longer the case so maybe union? But then it's harder to use.